src/request.ts

```
@@ -308,7 +318,9 @@ export class Request {
    */
   get length() {
     const len = this.get<string>('Content-Length');
-    if (len === '') return;
+    if (len === '') {
+      return;
```

Biome (1.9.4)

[error] This return should return a value because it is located in a getter.



(lint/suspicious/useGetterReturn)